### ISSUE TITLE
Newly added "old photos" for InstantUpload

### ISSUE THREAD

**Issue by sappsys** [owncloud/ios#710]
### Steps to reproduce
1. Go into an App (such as flickr)
2. save an image to the phone
3. photo does not get InstantUploaded

### Expected behaviour
Any new picture that gets saved to my camera roll should be uploaded
This picture is saved at the end of my camera roll, so the phone knows it's new

### Actual behaviour
The picture gets ignored. 

### Server configuration
**Web server:**
Apache 2.2.3

**Database:**
MySQL 5.0

**PHP version:**
5.5

### Client
**iOS version:**
9.3.1

**ownCloud app version:**
3.4.9

**Device model:**
Iphone 6S

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```
 
### Tell us what could be improved:
Because it's "creation date" is before the date of the last sync. The sync should look for "modification date".

**Comment by sappsys** [comment]
I'm not an IOS Coder at all, but looking at the code, I suspect it's around here 

                 PHFetchOptions *newAssetsFetchOptions = [PHFetchOptions new];
                 newAssetsFetchOptions.sortDescriptors = @[[NSSortDescriptor sortDescriptorWithKey:@"**creationDate**" ascending:YES]];
                 newAssetsFetchOptions.predicate = [NSPredicate predicateWithFormat:@"**creationDate** > %@", lastInstantUploadedAssetCaptureDate];
                 
                 PHFetchResult *newPhotos = [PHAsset fetchAssetsWithMediaType:PHAssetMediaTypeImage options:newAssetsFetchOptions];
                 
                 if (newPhotos != nil && [newPhotos count] != 0) {
                     
                     for (PHAsset *image in newPhotos) {
                         NSTimeInterval assetDateCreated = [image.creationDate timeIntervalSince1970];
                         if (assetDateCreated > ACTIVE_USER.timestampInstantUpload) {
                             ACTIVE_USER.timestampInstantUpload = assetDateCreated;
                             [ManageAppSettingsDB updateTimestampInstantUpload:assetDateCreated];
                         }
                     }

**Comment by Jon-Schneider** [comment]
The Instant Upload feature was actually recently rewritten, and that implementation will be coming out in 3.5.0. However, it will still demonstrate the same issue you mention.

The reason I went with creation date when I wrote the code is that using the modification date could have unforeseen consequences - for example, the app could literally reupload a photo if it is modified within the Photos app.
